app/code/Magento/Sales/Block/Adminhtml/Items/Column/Name.php

```diff
@@ -37,7 +37,7 @@ public function truncateString(
         string $value,
         int $length = 80,
         string $etc = '...',
-        string &$remainder = '',
+        ?string &$remainder = '',
```

Comment:
Not acceptable as PHP 7.0 is supported in 2.2.

Is this fixed in 2.3?

Reply:
In 2.3 it isn't in strict typing.